perdido/section-styles.js

```diff
@@ -74,11 +74,45 @@ window.styles = {
   fourUp: {
     background: '#FD7442',
     '& article': {
-        background: '#FD5C22',
-        extend: perdido.column('1/4'),
+      background: '#FD5C22',
+      extend: perdido.column('1/1'),
     },
   },
 
+  '@media (min-width: 500px)': {
+    fourUp: {
+      '& article': {
+        extend: perdido.column('1/2'),
+      }
+    }
+  },
+
+  '@media (min-width: 800px)': {
+    fourUp: {
+      '& article': {
+        extend: perdido.column('1/3'),
+      }
+    },
+    reordered: {
```

Comment:
@kof  For some reason this nested rule is generating a different selector than the other "reordered" key and not applying the styles like the fourUp one does.